db/migrate/20230824130328_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
Same here. As per the ER diagram, the association between the user and the comment is set up through the AuthorId and not directly on the User. You can use the same approach as you did for the posts.
![image](https://github.com/uno36/blog-app-rails/assets/33268927/2a2b4d08-50eb-41c5-ba51-37bb5ef5f528)